python/dgl/distributed/dist_dataloader.py

```diff
@@ -864,6 +869,7 @@ def __init__(self, g, eids, graph_sampler, device=None, **kwargs):
             else:
                 dataloader_kwargs[k] = v
 
+        collator_kwargs["gb_padding"] = 1
```

Comment:
this assignment of `1` looks weird. add detailed comment about why we specify it as `1`.

Reply:
Agree with Rhett-Ying

Reply:
Another question, why not set the default value of gb_padding to 1?